.ci/build_cli.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash -eE
+. /GIT/pushd_functions.sh
+. /GIT/ci_functions.sh
+pushd /GIT
+case $1 in
+    build)
+	configure
+        echo "Building NCCL sharp plugin"
```

Comment:
Fix formatting (e.g. you can use auto-formatting from PyCharm)

Reply:
Done